.github/workflows/pr-labeler.yaml

```
@@ -0,0 +1,12 @@
+name: pr-labeler
+on:
+  - pull_request
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          configuration-path: label-path-mapping.yaml
```

Let's put the file under `.github`.
Also, naming it `labeler.yml(yaml)` is recommended.
https://github.com/actions/labeler#create-githublabeleryml

```suggestion
          configuration-path: .github/labeler.yaml
```

---

Reflected by 
https://github.com/autowarefoundation/autoware.universe/pull/2266/commits/ce3bbd3f5b52de0cc1b7943448726ec499df539f